starters/deno-oak-denodb/src/db/resolvers/mutationHandler.ts

```diff
@@ -0,0 +1,38 @@
+import { Technologies } from '../model/technology.ts';
+export interface ITechnology {
```

Comment:
I'm not clear on why this interface is needed. Should it correspond to the GraphQL type `Technology`?

Reply:
This corresponds to the `TechnologyInput` in the type definitions.  
I can use `any` for now

Reply:
I see. In that case, I would suggest to name it that way: `TechnologyInput`. You could also place it in its own folder under the GraphQL API folder, e.g. `types`.

Which reminds me, ne other thing that we might consider is splitting the GraphQL-related code into its own folder. Right now it's located under the `db` folder, but the GraphQL code is really database-independent. What do you think about having a `graphql` folder, which would host API types like `TechnologyInput`, the schema, resolvers, etc.

Reply:
Can we generate those somehow?

If so, we should have this documented in the readme on how to do it. The initial types should be generated by us.

Reply:
For now, let's leave things as they are. 
I will be trying that in a different PR